lint/libs/lint-checks/src/main/java/com/android/tools/lint/checks/LogDetector.java

```diff
@@ -239,9 +239,14 @@ private static boolean checkWithinConditional(
             @NonNull UCallExpression logCall) {
         while (curr != null) {
             if (curr instanceof UIfExpression) {
-                UIfExpression ifNode = (UIfExpression) curr;
-                if (ifNode.getCondition() instanceof UCallExpression) {
-                    UCallExpression call = (UCallExpression) ifNode.getCondition();
+
+                UExpression condition = ((UIfExpression) curr).getCondition();
+                if (condition instanceof UQualifiedReferenceExpression) {
+                    condition = ((UQualifiedReferenceExpression) condition).getSelector();
```

Comment:
What will happen in case of `android.Log.isLoggable()`?
Looks like `getQualifiedChain()` or something similar is needed here.